HHS/Head-Start-TTADP
# [TTAHUB-1125] default select values

## Description of change

Change the default value of the goal picker selection component.

## How to test

1. Select a goal from the "Select recipient goal"
2. Complete the rest of the goal form
3. Click "Add new goal"
4. The goal selector should be displaying the "- Select -" placeholder instead of "Create new goal" by default.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1125


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
